src/main/resources/db/migration/courtcase/V109__make_fk_from_offence_to_court_case.sql

```diff
@@ -5,8 +5,14 @@ ALTER TABLE OFFENCE DROP COLUMN case_id;
 ALTER TABLE COURT_CASE DROP CONSTRAINT court_case_case_id_idempotent;
 
 -- Add columns to make the FK
-ALTER TABLE OFFENCE ADD COLUMN CASE_NO TEXT NOT NULL;
-ALTER TABLE OFFENCE ADD COLUMN COURT_CODE TEXT NOT NULL;
+ALTER TABLE OFFENCE ADD COLUMN CASE_NO TEXT;
```

Comment:
Correct me if I'm wrong but I don't think you'd need these steps in this instance because by the time you come to add the `NOT NULL` constraint at line 14 the table is already empty because of the delete on line 11?

Reply:
That's right. :-) What can I say - still early ?